k8-util/docker/fluvio/Makefile

```diff
@@ -0,0 +1,20 @@
+NAME = $(REGISTRY)/fluvio
+TAG = $(VERSION)
+BIN_DIR = ../../../target/x86_64-unknown-linux-musl/$(BIN_NAME)
+
+all: push
+
+copy_binaries:
+	mkdir -p target
+	cp	${BIN_DIR}/spu-server target
+	cp	${BIN_DIR}/sc-k8-server target
+
+build:	copy_binaries
+	docker build -f Dockerfile.fluvio  -t $(NAME):$(TAG)  ./target
+
+push:	build
+	docker push $(NAME):$(TAG)
```

Comment:
I think you need do perform [docker login](https://stackoverflow.com/questions/34434231/how-to-authenticate-for-docker-push) in order to publish on docker hub

Reply:
commit 14c0f2e adds a step for `docker login`, however it will require the following new secrets to be added:

`DOCKER_USERNAME` and `DOCKER_EMAIL`

Reply:
actually, you need to set password like here:  https://docs.docker.com/engine/reference/commandline/login/.

I created DOCKER_USERNAME and DOCKER_PASSWORD secret.

Reply:
Updated and squashed commits in b53b52f